src/Microsoft.Extensions.Configuration.Abstractions/ConfigurationExtensions.cs

```diff
@@ -26,17 +26,28 @@ public static string GetConnectionString(this IConfiguration configuration, stri
         /// </summary>
         /// <param name="configuration">The <see cref="IConfiguration"/> to enumerate.</param>
         /// <returns>An enumeration of key value pairs.</returns>
-        public static IEnumerable<KeyValuePair<string, string>> AsEnumerable(this IConfiguration configuration)
+        public static IEnumerable<KeyValuePair<string, string>> AsEnumerable(this IConfiguration configuration) => configuration.AsEnumerable(removePathFromChildKeys: false);
+
```

Comment:
I'm split, I like include or trim as the prefix, as opposed to use which doesn't really convey intent.

But dislike PathFromRoot, and prefer either Relative/Absolute Path.

includeAbsolutePath, trimAbsolutePath, makePathsRelative??